### PR TITLE
feat(tools): add missing files and class heritage

### PR DESCRIPTION
**Pull request by JulienChampagnol**

### CODE REVIEW

**Review by BotellaA — APPROVED** [review]

File: blueprints/blueprint_tools.py [hunk 1]
```diff
@@ -62,5 +92,77 @@ def upload_file():
...
+    for mandatory_file in mandatory_files:
+        mandatory_files_list.append(os.path.basename(mandatory_file))
+
+    additional_files_list = []
+    for additional_file in additional_files:
+        additional_files_list.append(os.path.basename(additional_file))
+
+    return flask.make_response(
+        {
+            "has_missing_files": has_missing_files,
```
Comment (BotellaA) [inline blueprints/blueprint_tools.py:140]: Key and value are identical. Is there a way like is JS to simplify dictionary creation? Juste out of curiosity

File: blueprints/tools/blueprint_crs_converter.py [hunk 2]
```diff
@@ -9,30 +10,18 @@
...
 
-with open("blueprints/tools/crs_converter_allowed_objects.json", "r") as file:
-    crs_converter_allowed_objects_json = json.load(file)
-
-with open("blueprints/tools/crs_converter_convert_file.json", "r") as file:
-    crs_converter_convert_file_json = json.load(file)
+crs_converter_routes = flask.Blueprint("crs_converter_routes", __name__)
+flask_cors.CORS(crs_converter_routes)
 
+with open("blueprints/tools/crs_converter_versions.json", "r") as file:
```
Comment (BotellaA) [inline blueprints/tools/blueprint_crs_converter.py:17]: Maybe a global question to answer all together : where should we import the json schema? At the beginning of each file or juste before its route definition?

**Review by BotellaA — APPROVED** [review]

File: blueprints/tools/blueprint_crs_converter.py (hunk 2 above)
Comment (JulienChampagnol) [reply blueprints/tools/blueprint_crs_converter.py:17]: I agree on this one

File: blueprints/blueprint_tools.py (hunk 1 above)
Comment (JulienChampagnol) [reply blueprints/blueprint_tools.py:140]: I looked it out, and it doesn't seem like it exists, there's discussions in the python community about implementing this, but not yet natively it seems

**Pull request merged**